packages/api/AlvTimeWebApi/Requests/PayoutRequest.cs

```diff
@@ -1,12 +1,11 @@
 ﻿using System;
-using AlvTimeWebApi.Validators;
+using AlvTime.Business.Validators;
 
 namespace AlvTimeWebApi.Requests;
 
 public class PayoutRequest
 {
     public DateTime Date { get; set; }
         
-    [HalfHour]
```

Comment:
Skulle ikke denne erstattes med QuarterHour? Ser ut som QuarterHour ligger på GenericPayoutHourEntry. Den trenger kanskje ikke ligge der hvis vi har den på PayoutRequest?

Reply:
Jeg valgte å ha beholder den i PayoutRequest fordi valideringen på timeregistreringen (CreateTimeEntryDto) også lå i Business-laget. Jeg kan godt flytte valideringen på Payout. Da flytter jeg i så fall begge deler. Hva tenker du blir best?

Reply:
Jeg er enig. Det gir mest mening å ha det på API request-klassen. Så vidt jeg husker, ville du ha det litt onion architecture/ clean architectur aktig i koden? Derfor: I og med at validering skal foregå på klasser i henholdsvis Business (CreateTimeEntryDto) og WebApi (PayoutRequest) fant jeg ikke ut av hvordan gjøre det uten å gjeninnføre QuarterHourValidering i WebApi prosjektet. Alternativet er å opprette en egen klasse for CreateTimeEntryDto i WebApi slik at valideringen kan legges der. Du kan bestemme:
Alt 1: En QuarterHour-validering i Business og en QuarterHour-validering i WebApi 
Alt 2: Opprette ny klasse i WebApi for validering med QuarterHour som da mappes til CreateTimeEntryDto
Hva høres best ut?